website/source/guides/web-ui/submitting-a-job.html.md

```diff
@@ -0,0 +1,185 @@
+quickly verifying how the job was configuring. Many properties from the job configuration will also
+be on the Job Overview page, but some deeper properties may only be available in the definition
+itself. It can also be convenient to see everything at once rather than traversing through task
+groups, allocations, and tasks.
+
+[![Job Definition][img-job-definition]][img-job-definition]
+
+## Job Versions
+
+From the subnav on any job detail page, you can access the Job Versions page.
+
+The Job Versions page will show a timeline view of every version of the job. Each version in the
+timeline includes the version number, the time the version was submitted, whether the version is/was
+stable, the number of changes, and the job diff itself.
+
+Reviewing the job diffs version by version can be used to debug issues in a similar manner to `git log`.
+
+[![Jobs Versions][img-job-versions]][img-job-versions]
+
+## Job Deployments
+
+From the subnav on any service job detail page, you can access the Job Deployments page.
+
+The Job Deployments page will show a timeline view of every deployment of the job. Each deployment
+in the timeline includes the deployment ID, the deployment status, whether or not the deployment
+requires promotion, the associated version number, the relative time the deployment started, and a
+detailed allocation breakdown.
+
+The allocation breakdown includes information on allocation placement, including how many canaries
+have been placed, how many canaries are expected, how many total allocations have been placed, how
+many total allocations are desired, and the health of each allocation.
+
+[![Jobs Deployments][img-job-deployments]][img-job-deployments]
+
+## Job Allocations
+
+From the subnav on any job detail page, you can access the Job Allocations page.
+
+The Job Allocations page will show a complete table of every allocation for a job. Allocations,
+being the unit of work in Nomad, are accessible from many places. The Job Overview page lists some
+of the recent allocations for a job for convenience and the Job Task Group page will list all
+allocations for that task group, but only the Job Allocations page shows every allocation across all
+task groups for the job.
+
+[![Jobs Allocations][img-job-allocations]][img-job-allocations]
+
+## Job Evaluations
+
+From the subnav on any job detail page, you can access the Job Evaluations page.
+
+The Job Evaluations page will show the most recent evaluations for the job. Evaluations are an
+internal detail of Nomad's inner scheduling process and as such are generally unimportant to
+monitor, but an experienced Nomad user can use evaluations to diagnose potential issues.
+
+[![Job Evaluations][img-job-evaluations]][img-job-evaluations]
+
+## Access Control
+
+Depending on the size of your team and the details of your Nomad deployment, you may wish to control
+which features different internal users have access to.
+
+Nomad has an access control list system for doing just that.
```

Comment:
Same feedback as other comment:

Maybe take this line out: **Nomad has an access control list system for doing just that.**

Instead, just add a sentence like this at the end to the previous paragraph:

**You can enforce this with Nomad's access control list system.**